pkg/helpers/tokencmd/basicauth_test.go

```diff
@@ -98,34 +98,6 @@ Password: `,
 			},
 		},
 
-		"interactive challenge": {
```

Comment:
@sallyom can this be changed to half-interactive when you pass username it should still ask you for a password, or we have one covering that case?

Reply:
this can be omitted, as the one above covers that, it's `"interactive challenge with default user"`

Reply:
(afaict :) )

Reply:
:+1: